recipes/libavif/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libavif" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AOMediaCodec/libavif/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: ea1603fc18e7dd20cf01f0b405156576886ecb5df84db8c0e87187cd2f8a00f4
+  patches:
+    - install-tests.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - nasm
+    - ninja
+    - meson
+  host:
+    - aom
+    # - svt-av1  # TODO: linking error with this optional library
+
+test:
+  source_files:
```

Comment:
Adding a test for the existence of built shared libraries would be good.
And header files, if there are any.